X2WOTCCommunityPromotionScreen/Src/X2WOTCCommunityPromotionScreen/Classes/CPS_UIArmory_PromotionHero.uc

```diff
@@ -544,18 +544,15 @@ simulated function UpdateNavHelp()
 	}
 
 	NavHelp = `HQPRES.m_kAvengerHUD.NavHelp;
+	if (NavHelp == none)
+		return; // Issue #28 - prevent 'accessed none' log warnings.
 
 	NavHelp.ClearButtonHelp();
 
 	if (UIAfterAction(Movie.Stack.GetScreen(class'UIAfterAction')) != none)
 	{
 		NavHelp.AddBackButton(OnCancel);
 
-		if (UIArmory_PromotionItem(List.GetSelectedItem()).bEligibleForPromotion && `ISCONTROLLERACTIVE)
-		{
-			NavHelp.AddSelectNavHelp();
```

Comment:
Can confirm this wasn't doing anything, although it raises the question of why the A button doesn't have a nav help item despite actually doing something. That should be a separate issue, though.